README.md

```diff
@@ -103,5 +103,9 @@ In order to test any of the quickbook features, you will need to create Intuit D
 6. Select "Accounting" and click "Create app"
 7. Click on "Keys"
 8. Copy the OAuth Key and Secret into your .env file. Use `.env.example` as a template.
+9. While logged into the application (Madeline site), visit Manage > Settings.
+10. Click on the button that says "Connect to Quickbooks."
+11. Sign in to the Quickbooks account for your developer account.
+12. Click authorize to connect your account data to Madeline.
```

Comment:
Note that it's better in Markdown to use `1.` for every item in a numbered list like this as it makes adding items in between easier. But this is fine for now.